search/highlight/format_html.go

```diff
@@ -47,18 +49,18 @@ func (a *HTMLFragmentFormatter) Format(f *Fragment, orderedTermLocations TermLoc
 			break
 		}
 		// add the stuff before this location
-		rv += string(f.Orig[curr:termLocation.Start])
-		// add the color
+		rv += html.EscapeString(string(f.Orig[curr:termLocation.Start]))
```

Comment:
nit pick: should `rv` be a `strings.Builder` instead?

Reply:
I'm going to open a new issue to track changing this to `strings.Builder`, as I don't want to hold up the fix any longer.